docs/source/dev_guide/pre_release_test_dev.rst

```diff
@@ -0,0 +1,63 @@
+***************************************
+Testing directions for making a release
+***************************************
+
+Use these directions to test a ``zppy`` dev environment prior to making a release candidate (RC).
+
+TODO: match up wording with the other doc
```

Comment:
Maybe make it all one doc with sub-steps A) dev, B) pre-release, C) testing Unified release